playbooks/insights_setup.yml

```diff
@@ -1,7 +1,22 @@
 ---
-# Placeholder playbook for setting up Insights services (advisor, drift, patch, vulnerability, and policies)
+# Steps required to get your system ready to use the Insights Client:
+# 1. Yum install the insights-client
+# 2. Register the insights-client
+# 3. Modify file permissions
 - hosts: localhost
+  become: yes
   tasks:
-    - name: Setup Insights
-      debug:
-        msg: "Insights setup"
+    - name: Update to the latest version of rhc
+      yum:
+        name: rhc
+        state: latest
+    - name: Update to the latest version of rhc-worker-playbook
+      yum:
+        name: rhc-worker-playbook
+        state: latest
```

Comment:
@subpop -  would we need to restart rhc after either the worker or rhc are upgraded?

Reply:
Yes, you should do a `systemctl daemon-reload` and a `systemctl restart rhcd` to start running the new code.

Reply:
It sounds like this might kill the playbook_worker.